src/main/java/com/cricketcraft/chisel/inventory/ContainerChisel.java

```diff
@@ -72,6 +72,14 @@ public ItemStack slotClick(int par1, int par2, int par3, EntityPlayer par4Entity
 		if (par3 == 2 && par2 == currentIndex)
 			return null;
 
+		// if the chisel was clicked, ignore the click
+		if (par1 >= 0) {
+			Slot slot = (Slot) this.inventorySlots.get(par1);
+			ItemStack stack = slot.getStack();
+			if (stack != null && stack.isItemEqual(chisel)) {
```

Comment:
Ahhhh - yes this fix breaks with 2 undamaged chisels of the same type. Thanks, I'll try and sort that.